sdk/search/Azure.Search.Documents/README.md

```diff
@@ -266,7 +266,52 @@ SearchOptions options = new SearchOptions
 };
 SearchResults<Hotel> response = client.Search<Hotel>("luxury", options);
 // ...
-``` 
+```
+
+### Creating an index
+
+You can use the `SearchServiceClient` to create a search index. Fields can be
+defined using convenient `SimpleField`, `SearchableField`, or `ComplexField`
+classes. Indexes can also define suggesters, lexical analyzers, and more.
```

Comment:
Is it worth linking to something like https://docs.microsoft.com/en-us/azure/search/search-what-is-an-index in here?